judgels-backends/uriel/uriel-api/src/main/java/judgels/uriel/api/contest/submission/bundle/ContestItemSubmissionData.java

```diff
@@ -8,7 +8,7 @@
 @JsonDeserialize(as = ImmutableContestItemSubmissionData.class)
 public interface ContestItemSubmissionData {
     String getContestJid();
-    String getProblemJid();
+    String getProblemAlias();
```

Comment:
Why change this? It is perfectly OK to be problemJid

Reply:
Reverted